svgdigitizer/__main__.py

```diff
@@ -80,16 +85,30 @@ def plot(svg, sampling_interval):
 )
 @click.argument("svg", type=click.Path(exists=True))
 def digitize(svg, sampling_interval):
+    r"""
+    Digitize a plot.
+
+    Produces a CSV from the curve traced in the SVG.
+
+    EXAMPLES::
+
+        >>> from .test.cli import invoke, TemporaryData
+        >>> with TemporaryData("**/xy_rate.svg") as directory:
+        ...     invoke(cli, "digitize", os.path.join(directory, "xy_rate.svg"))
+
+    """
     from svgdigitizer.svg import SVG
     from svgdigitizer.svgplot import SVGPlot
 
-    plot = SVGPlot(SVG(open(svg, "rb")), sampling_interval=sampling_interval)
+    with open(svg, "rb") as infile:
+        svg_plot = SVGPlot(SVG(infile), sampling_interval=sampling_interval)
+
     from pathlib import Path
 
-    plot.df.to_csv(Path(svg).with_suffix(".csv"), index=False)
+    svg_plot.df.to_csv(Path(svg).with_suffix(".csv"), index=False)
 
 
-@click.command()
+@click.command(name="cv")
```

Comment:
Does this set the cli argument name explicitly?
Is the `digitize_cv` also a possible argument or is it substituted by `cv`?

Reply:
No, the only possible name is `cv` from the command line. The problem here was that we are using `cv = ` in this function and it can be confusing since `cv` then totally changes its meaning.